doc/classes/SpringArm.xml

```diff
@@ -13,18 +18,21 @@
 			<argument index="0" name="RID" type="RID">
 			</argument>
 			<description>
+			Add objects to exclude from the collision check.
```

Comment:
Same here for indentation.

Method descriptions should also be in third person, so it should be something like "Adds an object to the list of objects excluded from the collision check."